conf.py

```diff
@@ -98,6 +99,24 @@
 #
 html_theme = 'sphinx_rtd_theme'
 
+html_title = 'EPICS Documentation'
+html_short_title = 'EPICS Docs'
+
+# Use RTD's canonical URL if available
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "/")
+
+html_show_sphinx = False
+html_copy_source = False
+html_permalinks_icon = '#'
```

Comment:
I just checked, and it seems with the ReadTheDocs theme, this option is ignored: the icon is forced to be 🔗